DxDispatch/cmake/d3d12.cmake

```
@@ -69,7 +69,7 @@ function(init_d3d12_cache_variables prefix)
 
     # <PREFIX>_D3D12_HEADERS_TAG
     set(${prefix}_D3D12_HEADERS_TAG
-        fb9a40f1b8165c848cf49fdb396722af7181ff97
+        v1.613.1
```

The recommendation is to use the SHA instead of the tag:
```suggestion
        de28d93dfa9ebf3e473127c1c657e1920a5345ee # v1.613.1
```

From https://cmake.org/cmake/help/latest/module/FetchContent.html:
"Where contents are being fetched from a remote location and you do not control that server, it is advisable to use a hash for GIT_TAG rather than a branch or tag name. A commit hash is more secure and helps to confirm that the downloaded contents are what you expected."